### ISSUE TITLE
CUDNN8 + CUDA 10.2 build?

### ISSUE THREAD

**Issue by oscarbg** [tensorflow/tensorflow#40423]
Hi,
similar to issue requesting a CUDA 11 build:
https://github.com/tensorflow/tensorflow/issues/40227
I think the most important of CUDA+CUDNN updates in relation to performance is CUDNN library updates..
seeing that CUDNN 8 ships with a library with CUDA 10.2 support and Tensorflow already supports CUDA 10.x, just asking if should be possible to integrate CUDNN 8 support faster than CUDA 11 and ship some nighlty build with CUDA 10.x+CUDNN 8 to test performance updates brought by this version..
thanks..

**Comment by ChipsSpectre** [comment]
Hey,
I just built Tensorflow 2.2 (default options build options from https://www.tensorflow.org/install/source, except of enabled GPU support).
On my system (RTX 2070, CUDA 10.2, CUDNN 8) the built library is correctly loaded and the results of an MNIST example training look reasonable.
```
2020-06-13 09:14:03.675559: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-06-13 09:14:03.821393: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.8
```
How could I add a nightly build for these CUDA/CUDNN versions to tensorflow?

**Comment by chsigg** [comment]
Hi Oscar, it's not really much less work to set up nightly releases for cuDNN 8 with CUDA 10.2 than going straight to CUDA 11. So I would rather do the latter, but it means that TF 2.3 will be released for CUDA 10.1 and cuDNN 7.

That said, the code is ready and you should be able to build TF for any combination of CUDA and cuDNN today. Is that an option for you?

**Comment by oscarbg** [comment]
Hi @chsigg,
good to know code is ready.. but lazy to build by myself.. will wait for TF 2.4 nightly builds, once 2.3 is branched, then this should be using CUDNN 8, right? I say because seems TF 2.3 branching seems coming next week anyway..
thanks..

**Comment by prova** [comment]
I was able to use these for CUDA 10.2 with cuDNN 7.6.5:  https://github.com/fo40225/tensorflow-windows-wheel.

**Comment by amahendrakar** [comment]
> good to know code is ready.. but lazy to build by myself.. will wait for TF 2.4 nightly builds,

@oscarbg,
The stable version of TF v2.4 is released it is compatible with CUDA 11.0 with cuDNN 8.0.

Could you please let us know if this is still an issue? Thanks!

**Comment by oscarbg** [comment]
it’s OK.. thanks..
